tests/suite.go

```diff
@@ -260,6 +275,10 @@ func isNotFound(err error) bool {
 	return err != nil && strings.Contains(err.Error(), "not found")
 }
 
+func alreadyExists(err error) bool {
+	return err != nil && strings.Contains(err.Error(), "already exists")
```

Comment:
perhaps `errors.Is` would work better here?

Reply:
It comes with custom object type. So once it is mixed into a string you can't determine it by internal function.

```
func IsAlreadyExists(err error) bool {
	return ReasonForError(err) == metav1.StatusReasonAlreadyExists
}
```